pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.jenkins-ci.plugins</groupId>
+		<artifactId>plugin</artifactId>
+		<version>2.26</version>
+	</parent>
+
+	<groupId>io.jenkins.blueocean</groupId>
+	<artifactId>jenkins-design-language</artifactId>
+	<version>0.0.124-SNAPSHOT</version>
+	<packaging>hpi</packaging>
+
+	<name>Jenkins Design Language</name>
+
+	<url>https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Design+Language</url>
+
+	<properties>
+		<node.version>6.4.0</node.version>
+		<npm.version>3.10.3</npm.version>
+	</properties>
+
+	<build>
+		<resources>
+			<resource>
+				<directory>.</directory>
+				<includes>
+					<include>package.json</include>
+					<include>dist/**/*</include>
+				</includes>
+			</resource>
+		</resources>
+		<plugins>
+			<!-- Disable InjectedTest -->
+			<plugin>
+				<groupId>org.jenkins-ci.tools</groupId>
+				<artifactId>maven-hpi-plugin</artifactId>
+				<configuration>
+					<disabledTestInjection>true</disabledTestInjection>
```

Comment:
I suspect this is because it is not a java/jelly plugin. Injected tests would do nothing?